src/utils/string/StringUtils.js

```diff
@@ -14,7 +14,7 @@ import {getAnonymousUserId} from '../users';
 import {getUserId} from '../token';
 
 const NO_POLL_UUID = '00000000-0000-0000-0000-000000000000';
-
+const urlRegex = /(https?:\/\/\S+)+/g;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `urlRegex` regular expression only matches URLs that start with "http://" or "https://". It might not match URLs that start with "www." or URLs without a protocol. Consider updating the regex to handle these cases.
```diff
- const urlRegex = /(https?:\/\/\S+)+/g;
+ const urlRegex = /(https?:\/\/|www\.)\S+/g;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->